palantirnet/butler
# Butler 3 global commands

Testing instructions:
* Set up a project following the readme
* See if it works


For deployment users will have to update the butler.defaults.js with their repo or we will have to add an override.js
